packages/AreaChart/src/AreaChart.tsx

```diff
@@ -139,7 +143,19 @@ const AreaChart: React.FunctionComponent<AreaChartProps> = ({
     const DEFAULT_SKELETON_WIDTH = 24;
```

Comment:
The constants could be put outside of the component. This way they won't get initialized on every render